kexec.c

```diff
@@ -108,6 +108,49 @@ static void add_kexec_segment(char *type, void *buf, unsigned long bufsize,
 	kexec_segment_nr++;
 }
 
+static GElf_Shdr *getshdr(Elf *e, int idx, GElf_Shdr *shdr)
+{
+	if (gelf_getshdr(elf_getscn(e, idx), shdr) == NULL) {
+		fprintf(stderr, "load_kernel_freebsd: get section failed\n");
```

Comment:
Should this still reference "load_kernel_freebsd"?

Reply:
Of course it shouldn't. Fixed.